api/src/main/java/io/strimzi/api/kafka/model/KafkaAuthorizationCustom.java

```diff
@@ -23,23 +26,24 @@
         builderPackage = Constants.FABRIC8_KUBERNETES_API
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({"type", "implementionClass", "superUsers"})
+@JsonPropertyOrder({"type", "authorizerClass", "config", "superUsers"})
 @EqualsAndHashCode
 public class KafkaAuthorizationCustom extends KafkaAuthorization {
     private static final long serialVersionUID = 1L;
 
     public static final String TYPE_CUSTOM = "custom";
 
-    private String implementionClass; 
+    private String authorizerClass;
     private List<String> superUsers;
+    private Map<String, Object> configProperties;
```

Comment:
why using a different name for the field and then the need for the `@JsonProperty(value = "config")` on the getter? I think that just `config` field sounds good imho.

Reply:
sure, I can change it. I was just trying to follow the pattern.

Reply:
done